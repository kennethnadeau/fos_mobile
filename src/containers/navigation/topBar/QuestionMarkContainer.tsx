import React from 'react';
import {View, StyleSheet} from 'react-native';
import {ScreenFC} from 'react-native-navigation-register-screens';
import {SCREENS} from '@fos/constants';
import {QuestionMarkIcon} from '@fos/components/navigation/topBar';

const QuestionMarkContainer: ScreenFC = () => {
  return (
    <View style={styles.container}>
      {/* FIXME Navigate to support screen */}
      <QuestionMarkIcon onPress={() => {}} />
    </View>
  );
};

QuestionMarkContainer.screenName = SCREENS.COMPONENTS.TOP_BAR.QUESTION_MARK;

const styles = StyleSheet.create({
  container: {
    flex: 1,
  },
});

export default QuestionMarkContainer;
