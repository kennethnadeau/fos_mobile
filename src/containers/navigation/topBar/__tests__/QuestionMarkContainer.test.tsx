import React from 'react';
import {render, RenderAPI} from '@fos/utils/testing';
import QuestionMarkContainer from '../QuestionMarkContainer';

describe('<QuestionMarkContainer />', () => {
  let questionMarkContainer: RenderAPI;

  beforeEach(() => {
    questionMarkContainer = render(<QuestionMarkContainer componentId="id" />);
  });

  it('should match snapshot', () =>
    expect(questionMarkContainer.toJSON()).toMatchSnapshot());
});
