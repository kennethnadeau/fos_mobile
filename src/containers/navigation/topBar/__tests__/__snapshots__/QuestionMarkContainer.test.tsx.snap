// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QuestionMarkContainer /> should match snapshot 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <SvgMock
        accessibilityLabel="Support"
        accessible={true}
        fill="white"
        height={64.28571428571428}
        width={64.28571428571428}
      />
    </View>
  </View>
</RNCSafeAreaProvider>
`;
